driver-examples/tycho/com.datastax.driver.examples.rcp.repository/pom.xml

```diff
@@ -0,0 +1,102 @@
+<!--
+
+         Copyright (C) 2012-2015 DataStax Inc.
+
+      Licensed under the Apache License, Version 2.0 (the "License");
+      you may not use this file except in compliance with the License.
+      You may obtain a copy of the License at
+
+         http://www.apache.org/licenses/LICENSE-2.0
+
+      Unless required by applicable law or agreed to in writing, software
+      distributed under the License is distributed on an "AS IS" BASIS,
+      WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+      See the License for the specific language governing permissions and
+      limitations under the License.
+
+-->
+<project
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0
+    http://maven.apache.org/xsd/maven-4.0.0.xsd"
+    xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>com.datastax.cassandra</groupId>
+  <artifactId>com.datastax.driver.examples.rcp.repository</artifactId>
+  <version>0.0.1-SNAPSHOT</version>
+  <name>DataStax Java Driver for Apache Cassandra Examples - Eclipse RCP - Local Repository</name>
+  <packaging>pom</packaging>
+
+  <properties>
+    <driver.version>2.1.5</driver.version>
+    <guava.version>18.0</guava.version>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+  </properties>
+
+  <build>
+
+    <plugins>
+
+      <plugin>
```

Comment:
I finally gave up and started used p2-maven-plugin. It is way more flexible, I must admit. In particular, it is the only way to pull the test-jar dependency of the driver.